### ISSUE TITLE
TypeError: 'Pointer' object is not subscriptable with crispy-forms>=2.0

### ISSUE THREAD

**Issue by estyxx** [sveetch/crispy-forms-foundation#50]
I'm getting this error with 

django-crispy-forms = "^2.0"
crispy-forms-foundation = "^0.9.0"

```python
Traceback (most recent call last):
  File "/app/tbxcodingtask/tracker/test/test_torchbox.py", line 16, in test_create_project
    response = client.get(
               ^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/test/client.py", line 836, in get
    response = super().get(path, data=data, secure=secure, **extra)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/test/client.py", line 424, in get
    return self.generic(
           ^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/test/client.py", line 541, in generic
    return self.request(**r)
           ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/test/client.py", line 810, in request
    self.check_exception(response)
  File "/usr/local/lib/python3.11/site-packages/django/test/client.py", line 663, in check_exception
    raise exc_value
  File "/usr/local/lib/python3.11/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/views/generic/base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/views/generic/base.py", line 142, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/views/generic/edit.py", line 180, in get
    return super().get(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/views/generic/edit.py", line 144, in get
    return self.render_to_response(self.get_context_data())
                                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/views/generic/edit.py", line 74, in get_context_data
    kwargs["form"] = self.get_form()
                     ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/views/generic/edit.py", line 39, in get_form
    return form_class(**self.get_form_kwargs())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/tbxcodingtask/tracker/forms.py", line 14, in __init__
    super(BaseTrackerForm, self).__init__(*args, **kwargs)
  File "/usr/local/lib/python3.11/site-packages/crispy_forms_foundation/forms.py", line 193, in __init__
    self.init_helper()
  File "/usr/local/lib/python3.11/site-packages/crispy_forms_foundation/forms.py", line 112, in init_helper
    if isinstance(self.fields[pointer[1]].widget,
```
because `pointer` seems to be an object `Pointer(positions=[1], name='title')` 

yes they introduced here: https://github.com/django-crispy-forms/django-crispy-forms/pull/1293/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

**Comment by sveetch** [comment]
Not sure this would be the only issue with crispy 2.x since they removed bootstrap and it needs to be installed apart, but crispy-foundation rely on some layout objects from bootstrap theme.

**Comment by sveetch** [comment]
This issue should have been fixed in v1.0.0
